metabase/metabase
# Merge events by pixel coordinates

Epic: https://github.com/metabase/metabase/issues/20289

How to test:
- Create events that close to each other. The ones on the screenshot are Jan 1 and Jan 24.
- Resize the chart and make them group / ungroup with each other
- Note that the position of the event line & icon is the average of events within the group

<img width="1220" alt="Screenshot 2022-03-28 at 18 21 55" src="https://user-images.githubusercontent.com/8542534/160431888-fe4462c0-517a-401c-9304-169cac966d52.png">
<img width="1183" alt="Screenshot 2022-03-28 at 18 22 11" src="https://user-images.githubusercontent.com/8542534/160431914-09628b3f-5b62-4a7b-a96a-ff7c554af460.png">
<img width="1182" alt="Screenshot 2022-03-28 at 18 22 22" src="https://user-images.githubusercontent.com/8542534/160431922-bc07f547-f359-452b-b27a-dc472351d64a.png">
